python/llm/example/transformers/transformers_int4/transformers_int4_pipeline.py

```diff
@@ -47,7 +47,7 @@
         print('Output:', output_str)
         print(f'Inference time: {end-st} s')
     elif model_path == 'THUDM/chatglm-6b':
-        model = AutoModel.from_pretrained(model_path, trust_remote_code=True, load_in_4bit=True)
+        model = AutoModel.from_pretrained(model_path, trust_remote_code=True, load_in_4bit=True, keep_in_fp32_modules=['output_layer'])
```

Comment:
layers_not_to_convert=['output_layer']

Reply:
actually change `keep_in_fp32_modules` to `layers_using_high_precision`

Reply:
> actually change `keep_in_fp32_modules` to `layers_using_high_precision`

I think this PR is now unnecessary, reasons can be found [here](https://github.com/analytics-zoo/nano/issues/434#issuecomment-1635201788)